src/app/components/Book/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '@constants/colors';
+import { SIZES } from '@constants/fonts';
+
+export const styles = StyleSheet.create({
+  container: {
+    backgroundColor: COLORS.WHITE,
+    width: 375,
+    height: 102,
```

Comment:
In mobile, we don't set the `width` and `height` in most cases. For this, you must use `margin (space between components)` and `padding (space inside components)`. Also, you can use `width: 100%` to fit all the screen.